src/app/story/directives/status.component.ts

```diff
@@ -26,6 +26,9 @@ import { StoryModel } from '../../shared';
       <dt *ngIf="isPublished && !isScheduled">Published</dt>
       <dd *ngIf="isPublished"><p>{{story.publishedAt | date:"short"}}</p></dd>
 
+      <dt *ngIf="isReleased">Released</dt>
```

Comment:
Maybe use "Release" instead of "Released", since any episode showing this will not actually be published?  (Also matches the "Release Date" language at the bottom of the form).